File: datasets/File06_Old.java
// File06_Old.java
public class File06 {
    private String userName; // Variable to be renamed

    public File06(String name) {
        this.userName = name;
    }

    public String getUserName() { // Method to be renamed
        return this.userName;
    }

    public void displayUserInfo() {
        System.out.println("User: " + getUserName());
    }
}

File: python-lhdiff-project/datasets/Aakanksha/File06_Old.py
# File06_Old.py

class File06:
    def __init__(self, name):
        self.user_name = name  # Variable to be renamed

    def get_user_name(self):  # Method to be renamed
        return self.user_name

    def display_user_info(self):
        print("User:", self.get_user_name())